rosetta-source/src/main/rosetta/product-asset-type.rosetta

```diff
@@ -52,20 +52,22 @@ type CreditDefaultPayout extends PayoutBase: <" The credit default payout specif
 		priceQuantity exists
 
 type GeneralTerms: <" A class specifying a set of non-monetary terms for the Credit Derivative Transaction, including the buyer and seller and selected items from the ISDA 2014 Credit Definition article II, such as the reference obligation and related terms. The CDM GeneralTerms class corresponds to the FpML GeneralTerms complex type, except that the effectiveDate and scheduledTerminationDate have been positioned as part of the InterestRatePayout class in the CDM instead of in GeneralTerms.">
+	
+    referenceInformation ReferenceInformation (0..1) <"This attribute contains all the terms relevant to defining the reference entity and reference obligation(s).">
+    indexReferenceInformation IndexReferenceInformation (0..1) <"This attribute contains all the terms relevant to the underlying Index.">
+    additionalIndexReferenceInformation AdditionalIndexReferenceInformation (0..1) <"This attribute contains all the additional terms relevant to defining the Credit DefaultSwap Index.">
 
```

Comment:
why is indexReferenceInformation being explicitly added as an attribute? surely if an index is specified then that information will be inside "index"

Reply:
'indexReferenceInformation' is already present here. In this change, the 'indexReferenceInformation' in split into 'indexReferenceInformation' and 'additionalIndexReferenceInformation'. 
We've just added 'additionalIndexReferenceInformation' in the type above.